arbnode/node.go

```diff
@@ -948,6 +948,7 @@ func createNodeImpl(
 		if err != nil {
 			return nil, err
 		}
+		log.Info("running as validator", "smartContractWallet", wallet.Address(), "strategy", config.Validator.Strategy)
```

Comment:
This is the smart contract wallet address, right? What will this do if one hasn't been created yet?

Reply:
The log is right after call to `validator.NewValidatorWallet`, so `wallet` is definitely not nil. nil,

Reply:
And log.Info properly prints `nil` if `wallet.Address()` is nil